### PR TITLE
Existing neighbourhood record updates

### PR DESCRIPTION
**Pull request by stepps00**
Fixes a portion of https://github.com/whosonfirst-data/whosonfirst-data/issues/1695.

There will be a second round of "new" neighbourhood records coming in, once this PR is merged. Over to @nvkelso for  👀 .

### CODE REVIEW

File: data/858/982/75/85898275.geojson [hunk 1]
```diff
@@ -5,7 +5,7 @@
     "edtf:cessation":"uuuu",
     "edtf:inception":"uuuu",
     "geom:area":0.0,
-    "geom:area_square_m":21511309.214026,
+    "geom:area_square_m":0.0,
```
Comment (nvkelso) [inline data/858/982/75/85898275.geojson]: What happened to the area?!

Comment (nvkelso) [reply data/858/982/75/85898275.geojson]: Perhaps **exportify** was leaving stale areas around, and new exportify doesn't? At least it's consistent with `geom:area` now.

Comment (stepps00) [reply data/858/982/75/85898275.geojson]: Looks like this is now Point geometry that incorrectly had a `geom:area_square_m` value larger than `0.0` (presumably from a bad exportify?). This change exportified and correctly updated the `geom:area_square_m` value to `0.0`... so I think we're good here.

File: data/858/983/01/85898301.geojson [hunk 2]
```diff
@@ -30,6 +30,9 @@
     "name:deu_x_variant":[
         "Grossfeldsiedlung"
     ],
+    "name:ger_x_preferred_x_variant":[
```
Comment (nvkelso) [inline data/858/983/01/85898301.geojson]: This should be `deu`, looks like we'll need to remap it from the input?

File: data/858/982/97/85898297.geojson [hunk 3]
```diff
@@ -37,6 +37,9 @@
     "name:eng_x_preferred":[
         "Gro\u00dfer Biberhaufen"
     ],
+    "name:ger_x_preferred_x_variant":[
```
Comment (nvkelso) [inline data/858/982/97/85898297.geojson]: Same as below, this language code should be `deu` not `ger`.

File: data/858/984/01/85898401.geojson [hunk 4]
```diff
@@ -38,6 +38,9 @@
     "name:eng_x_variant":[
         "Suessenbruenn"
     ],
+    "name:ger_x_preferred_x_variant":[
```
Comment (nvkelso) [inline data/858/984/01/85898401.geojson]: This property name is 2x jumbled.

File: data/858/983/01/85898301.geojson (hunk 2 above)
Comment (nvkelso) [reply data/858/983/01/85898301.geojson]: Please audit the language codes used in the SHP's DBF and make sure there's not any others we should remap.

File: data/858/982/97/85898297.geojson (hunk 3 above)
Comment (stepps00) [reply data/858/982/97/85898297.geojson]: Good catch.. the script has a iso_lookup table, but it's looking like my logic isn't respecting it.

File: data/858/983/01/85898301.geojson (hunk 2 above)
Comment (stepps00) [reply data/858/983/01/85898301.geojson]: Ill update in a new commit.

File: data/858/984/01/85898401.geojson (hunk 4 above)
Comment (stepps00) [reply data/858/984/01/85898401.geojson]: This should be fixed in the most recent commit.

**Comment by stepps00** [comment]
The most recent commit updates logic to remove bad `name:` properties and deduplicates name values across `name:` properties. Ready for another :eyes:

File: data/858/982/97/85898297.geojson [hunk 5]
```diff
@@ -32,7 +32,8 @@
         "Gro\u00dfer Biberhaufen"
     ],
     "name:deu_x_variant":[
-        "Gr. Biberhaufen"
+        "Gr. Biberhaufen",
+        "Gro\u00dfer Biberhaufen"
```
Comment (nvkelso) [inline data/858/982/97/85898297.geojson]: This variant dup's the preferred name, and should be removed?

Comment (stepps00) [reply data/858/982/97/85898297.geojson]: Added another commit, this should be fixed.

**Comment by stepps00** [comment]
The changes in this PR should now address all comments.

**Pull request merged**